index.html/site.js

```diff
@@ -1,103 +1,109 @@
 const eEqualsMCsquared = {
-    equation: 'E = mc²',
-    description: 'Mass-energy equivalence equation',
-    constants: {
-      speedOfLight: 299792458, // Speed of light in meters per second never changes
-    },
-  
-    calculateEnergy: function (mass) {
-      return mass * this.constants.speedOfLight ** 2;
-    },
-  
-    units: {
-      mass: 'kg',
-      energy: 'Joules', // Self-explanatory
-    },
-  
-    scientificContext: {
-      physicist: 'Albert Einstein',
-      year: 1905,
-      theory: 'Special Relativity', // Some info from notes
-    },
-  
-    relatedConcepts: ['Nuclear Energy', 'Particle Physics'],
-    additionalDetails: {
-      significance: 'E=mc² revolutionized our understanding of energy and mass.',
-      howItHelps: [
-        'Nuclear energy generation',
-        'Medical imaging (PET scans)',
-        'Space exploration',
-      ],
-    },
-  
-    examples: {
-      example1: {
-        mass: 1, // 1 kg
-        energy: this.calculateEnergy(1), // Calculate energy for 1 kg of mass
-      },
-  
-      example2: {
-        mass: 10,
-        energy: this.calculateEnergy(10),
-      },
-  
-      example3: {
-        mass: 50,
-        energy: this.calculateEnergy(50),
-      },
-  
-      example4: {
-        mass: 100,
-        energy: this.calculateEnergy(100),
-      },
-
-      example5: {
-          mass: 500,
-          energy: this.calculateEnergy(500),
```

Comment:
- The examples aren't related to the rest of object so I removed them, we'll put them somewhere else
- `this` is a pretty tricky concept to grasp but basically since you're using it as an object inside and object it was breaking your code here